pkg/action/uninstall.go

```diff
@@ -227,23 +168,20 @@ func (u *Uninstall) deleteRelease(rel *release.Release) (kept string, errs []err
 	}
 
 	filesToKeep, filesToDelete := filterManifestsToKeep(files)
+	var kept string
 	for _, f := range filesToKeep {
 		kept += f.Name + "\n"
 	}
 
+	var builder strings.Builder
 	for _, file := range filesToDelete {
-		b := bytes.NewBufferString(strings.TrimSpace(file.Content))
-		if b.Len() == 0 {
-			continue
-		}
-		if err := u.cfg.KubeClient.Delete(b); err != nil {
-			u.cfg.Log("uninstall: Failed deletion of %q: %s", rel.Name, err)
-			if err == kube.ErrNoObjectsVisited {
-				// Rewrite the message from "no objects visited"
-				err = errors.New("object not found, skipping delete")
-			}
-			errs = append(errs, err)
-		}
+		builder.WriteString("\n---\n" + file.Content)
```

Comment:
One more note here. If we do not like glueing manifests back together like this, there are 2 alternatives:
1. Use `KubeClient.Build` for each manifest and then put those objects together into one slice
2. Refactor the filtering logic to filter a `ResourceList` instead of a raw manifest